checks/check122

```diff
@@ -42,9 +42,9 @@ check122(){
         textFail "$REGION: Policy $policy allows \"*:*\"" "$REGION" "$policy"
       done
     else
-        textPass "$REGION: No custom policy found that allow full \"*:*\" administrative privileges" "$REGION" "$policy"
+        textPass "$REGION: No custom policy found that allow full \"*:*\" administrative privileges" "$REGION"
```

Comment:
Why do you remove policy here and you don´t in textFail? it would brake the csv anyway.

Reply:
Reading the discord chat, I understand that point but why not to sanitize the variable `$policy` to make sure it doesn't  include a comma? 

Something like `echo $policy | awk -F',' '{ print $1 }'`

Reply:
having resource name in any status helps or will help for visualizations

Reply:
If you look at the for loop, you will issue a textPass once to tell that no policy contains '*:*', so it is not relevant to add the latest policy checked, or you should have one textPass per policy and include the PolicyName.
Regarding the textFail, it triggers for each policy that fail and the variable policy is sanitized in this case.

Reply:
got it